src/schema/artist/carousel.ts

```diff
@@ -55,7 +55,7 @@ const ArtistCarousel: GraphQLFieldConfig<{ id: string }, ResolverContext> = {
           })
           .then(showsWithImages => {
             return showsWithImages.concat(
-              removeReproductionsFromArtworks(artworks)
+              artworks
                 .slice(0, 6) // Always return the top 7 artworks
```

Comment:
Since we only wanted to return the top 7 iconic works (and thus overfetched to 10 b/c of the filter), we can switch to only requesting 7 (instead of 10) a few lines above. Save some bytes!

Reply:
Good call!! 💯 

Fixed with: 22a00c9